client/src/api/eventstream.tsx

```diff
@@ -165,6 +162,14 @@ export const handleStreamedEvents = (
     });
   }
 
+  for (const connection of events.updated_iads) {
+    dispatch(updateIadsConnection(connection as IadsConnection));
+  }
+
+  for (const id of events.deleted_iads) {
+    dispatch(removeIadsConnection(id as string));
+  }
+
```

Comment:
All changes in this file should be obvious-ish, we're trying to decouple it from TGOs and listen for update or delete events. Initialization is done somewhere else and I notice no issues so I didn't touch that part.